src/views/local-token-service.handlebars

```diff
@@ -123,6 +123,15 @@ let text = client.get("<span class="origin">https://gateway.local</span>/things"
         </div>
       </div>
     </section>
+    <script>
+/* postToken to client apps if any */
+if (config.get('oauthPostToken') && (window.location.protocol === "file:") {
+  window.addEventListener("message", function(ev) {
+    if (ev.data.message === "token") {
+      ev.source.postMessage({ message: { token: "{{token}}" }}, "*");
+    }
+});
```

Comment:
The indentation of this is a bit off and the close brace from the initial `if` statement is missing

Reply:
fixed